src/actions/commands/actions.ts

```diff
@@ -1183,7 +1185,16 @@ abstract class CommandFold extends BaseCommand {
   }
 
   public async exec(position: Position, vimState: VimState): Promise<void> {
-    await vscode.commands.executeCommand(this.commandName);
+    let timesToRepeat = vimState.recordedState.count || 1;
+    if (this.direction !== '') {
+      await vscode.commands.executeCommand(this.commandName, {
+        levels: timesToRepeat,
+        direction: this.direction,
+      });
+    } else {
+      await vscode.commands.executeCommand(this.commandName);
+    }
```

Comment:
would be interested if this can be done with a neat one liner?

Reply:
```typescript
const args = this.direction != '' ? { levels: timesToRepeat, direction: this.direction} : undefined;
await vscode.commands.executeCommand(this.commandName, args);
```
seems like the nicest way of writing this

Reply:
linter turned that into 4 lines :D